containers/Auth/AuthProvider.js

```diff
@@ -39,6 +39,11 @@ export const AuthProvider = (props) => {
       AsyncStorage.setItem('authToken', authToken);
       setState({ ...state, loading: false, token: authToken });
     },
+    signOut: async () => {
+      console.log('I was triggered during componentDidMount123');
```

Comment:
we should remove this and above as well

Reply:
Alread did that here https://github.com/liqd/zt-app/pull/78/commits/2d79284b6a28c82a8478682229367eb9080bd96a